README.md

```diff
@@ -36,6 +36,20 @@ var meisterPlayer = new Meister('#player', {
 });
 ```
 
+
+#### autoHide *[Boolean]* ####
+
+Set autoHiding on/off. When autoHiding is used the watermark will slowly fade out (8 seconds)
```

Comment:
Would be nice if we could make this configurable in the future, but I see how this is a bit of a hassle now with the css rule.

Reply:
v9.0 ;)